pkg/resolver/search.go

```diff
@@ -121,6 +122,65 @@ func (s *SearchResult) Items() ([]map[string]interface{}, error) {
 	return r, e
 }
 
+// HasNextPage returns true if there are more results available after the current page.
+// Calculated using: (offset + limit) < count
+func (s *SearchResult) HasNextPage() (*bool, error) {
+	if !s.matchesManagedHubFilter() {
+		result := false
+		return &result, nil
+	}
+
+	// Get the total count
+	count, err := s.Count()
```

Comment:
This triggers a Postgres query like `SELECT count(*) ...` 
This query could be redundant if we are also resolving the s.Items.
count = len(s.Items())

It gets a little complex to coordinate the different resolvers. So, I'm thinking it's easier to keep this calculation in the client.

Reply:
I am removing the HasPreviousPage , HasNextPage